.lldbinit

```diff
@@ -0,0 +1,2 @@
+command regex _pp 's/(.+)/e -l swift -o -- Pretty.prettyPrint(%1)/'
```

Comment:
I have a question that what happen when evaluation expression:

```
(lldb) _p 1 + 1 // This is success or failed?
```

In other words, what dose `%1` means?

Reply:
It's working!
This is because This %1 substitute the string meeting regex of  `(. +)`
![image](https://user-images.githubusercontent.com/14083051/102161986-b2900c00-3ecb-11eb-8fe5-22a24f425fbd.png)

For example, you can make this 
```
(lldb) command regex compare 's/(.+) (.+)/po %1 == %2/'
(lldb) compare 122 125
-> false

(lldb) compare 122 122
-> true
```

Reply:
I fully understood!